Assignment/starter_code/js/app.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function(){
+	alert('hey')
+
+	var city = $('#city-type').val();
+	if (city==='NY'||'New York'||'NYC'){
```

Comment:
Syntax! You're writing your own syntax to the computer, this makes sense in english because its read city equals NY or New York or NYC but it makes no sense to a computer.  Remember, we're not programming to siri - never assume the computer knows anything. The proper syntax is ```if (city === 'NY' || city === 'New York' || city === 'NYC') {```